tests/integration/routes/accounts/get-accounts-test.js

```diff
@@ -54,8 +55,20 @@ getServer(function (error, server) {
       t.end()
     })
 
-    group.test('Not an admin', {todo: true}, function (t) {
-      t.end()
+    group.test('Not an admin', function (t) {
+      var requestOptions = _.defaultsDeep({
+        headers: {
+          authorization: 'Bearer cGF0LWRvZToxMjc1MDA6zEZsQ1BuO-W8SthDSrg8KXQ8OlQ'
```

Comment:
cool @gr2m I'll try to do that tonight (france time). In general, I find the code has very few comments. Especially regarding the handling of the errors... I'm going to comment on the file I changed with another question for you...